roles/splunk/tasks/configure_idxc_member.yml

```
@@ -6,12 +6,21 @@
 # https://docs.splunk.com/Documentation/Splunk/latest/Indexer/ConfigurepeerswithCLI
 # Splunk version < 8.1 supports mode=slave
 # https://docs.splunk.com/Documentation/Splunk/8.0.9/Indexer/ConfigurepeerswithCLI
-- name: Setting clustering mode based on Splunk version number
+
+- name: Setting clustering mode to slave and uri to master base on Splunk version
+  set_fact:
+    mode_value: "slave"
```

And this:
```suggestion
    mode_value: "{% if splunk_version_release | version('8.1', '<') %}slave{% else %}peer{% endif %}"
```
These changes (if tested and working) will combine both 8.1 and later into one task